workbench-core/example/app/src/configs/staticRouteConfig.ts

```diff
@@ -72,8 +72,32 @@ export const routesMap: RoutesMap = {
         action: 'CREATE',
         subject: 'DatasetAccess'
       }
+    ],
+    GET: [
+      {
+        action: 'READ',
+        subject: 'DatasetAccess'
+      }
     ]
   },
+  [`/datasets/${dataSetPrefix.toLowerCase()}-${uuidRegExpAsString}/permissions/users/${uuidRegExpAsString}`]:
+    {
+      GET: [
+        {
+          action: 'READ',
+          subject: 'DatasetAccess'
+        }
+      ]
+    },
+  [`/datasets/${dataSetPrefix.toLowerCase()}-${uuidRegExpAsString}/permissions/roles/${groupIDRegExpAsString}`]:
```

Comment:
just making sure this is right: says `roles` but its `groupIDRegExpAsString` and i didn't know those were interchangable?

Reply:
it's the example app - do not think the necessarily has any bearing on choices made in SWB or any other consumers. That said, I chose roles to match existing verbiage on the user management routes.

Reply:
oh. That's a GroupID as in an SWB projectGroup?  Yeah, um, turns out it works for our purposes! I think it's because we are linked to Cognito and using the Groupid as the role name.